src/action.rs

```diff
@@ -25,6 +25,7 @@ use std::{
 ///    2. `Action::Terminate` indicates to `Core` that no new actions should be taken and all
 ///       pending events should be handled.
 ///       After completion `Core` will send `Event::Terminated`.
+#[allow(clippy::large_enum_variant)]
```

Comment:
Just one thing I meant to ask before approving: did you update the safe_vault code and run its integration tests?

Reply:
That should really be a case of us using semver properly and the consumer to update properly.

Reply:
Just ran the tests. All passed.